specs/waku/v2/waku-v2-rpc-api.md

```diff
@@ -189,26 +192,29 @@ The following structured types are defined for use on the Store API:
 #### StoreResponse
 
 `StoreResponse` is an `Object` containing the following fields:
+
 | Field | Type | Inclusion | Description |
-| ----: | :--: | :--: |----------- |
+| ----: | :---: | :---: |----------- |
 | `messages` | `Array`[[`WakuMessage`](#WakuMessage)] | mandatory | Array of retrieved historical messages |
-| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#`get_waku_v2_store_v1_messages`) | Paging information from which to resume further historical queries |
+| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#get_waku_v2_filter_v1_messages) | Paging information from which to resume further historical queries |
```

Comment:
```suggestion
| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#get_waku_v2_store_v1_messages) | Paging information from which to resume further historical queries |
```
Ah, thanks, I see this link was broken due to the backticks, but it should indeed point to `get_waku_v2_store_v1_messages` where the conditions for including `pagingOptions` are discussed.